File: Programlar/Task 2 (Adlı alqoritmlər)/Kvadrat tənlik.cpp
#include<iostream>
#include<cmath>
using namespace std;
int main()
{
	float a,b,c;
	cin>>a>>b>>c;
	if(a==0 && b==0 && c==0)cout<<"sonsuz sayda kökü var";
	else if(a==0 && b==0)cout<<"kökü yoxdur";
	else if (a==0)cout<<-c/b;
	else{
		float D = b*b-4*a*c;
		if(D>0){
			float x1 = (-b + sqrt(D))/(2*a);
			float x2 = (-b - sqrt(D))/(2*a);
			cout<<x1<<" "<<x2;
		}else if(D==0){
			float x = (-b + sqrt(D))/(2*a);
			cout<<x;
		}else{
			cout<<"kökü yoxdur";
		}
	}
}

File: Programlar/Task 2 (Adlı alqoritmlər)/Kvadrat tənlik.py
from math import sqrt
a = float(input("a= "))
b = float(input("b= "))
c = float(input("c= "))
if a==0 and b==0 and c==0:
    print("sonsuz sayda koku var")
elif a==0 and b==0:
    print("koku yoxdur")
elif a==0:
    print(-c/b)
else:
    D = b**2-4*a*c
    if D>0:
        x1 = (-b + sqrt(D))/(2*a)
        x2 = (-b - sqrt(D))/(2*a)
        print(x1,x2)
    elif D==0:
        x = (-b + sqrt(D))/(2*a)
        print(x)
    else: print("koku yoxdur")
